src/harbour-tuschbox.cpp

```diff
@@ -53,12 +53,16 @@ int main(int argc, char *argv[])
 
     qmlRegisterType<QQuickFolderListModel>("harbour.tuschbox.Tuschbox", 1, 0, "FolderListModel");
 
+
     QQuickView *view = SailfishApp::createView(); // I get a white background with this.
     view->setSource(SailfishApp::pathTo("qml/harbour-tuschbox.qml"));  // So I do this ;)
 
     FM *fileAction = new FM();
     view->engine()->rootContext()->setContextProperty("_fm", fileAction);
 
+    Settings *settings = new Settings();
```

Comment:
Ok I thought about it after reading into it. As this particular setting is non user editable directly (e.g. not a setting the user has to set) we can save it separately.  
Other settings I like to keep in the database as it makes it easier to migrate whole configurations later on by just copying one folder/file. (instead of two)